test/k8sT/manifests/cilium-cm-patch-clean-cilium-state.yaml

```diff
@@ -25,3 +25,5 @@ data:
   enable-ipv6: "true"
 
   preallocate-bpf-maps: "true"
+  bpf-ct-global-tcp-max: "524288"
```

Comment:
we don't need this here as these are the default values picked from the default k8s descriptor.